src/hooks/useServicesList.js

```diff
@@ -0,0 +1,17 @@
+import { useEffect, useState } from "react";
+
+import { GetServices } from "../services/ApiService";
+
+const useServicesList = () => {
+  const [serviceItems, setServiceItems] = useState([]);
```

Comment:
this hook should also include isLoading. Then when it's used you can displaying a loading message to validate this test, https://github.com/baltimorecounty/react-services-list/pull/5/files#diff-019a00e024a07fb5605b598fc002f4b8R42.

Reply:
done